fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -3154,11 +3154,20 @@ public void addPartitions(Database db, String tableName, List<SingleRangePartiti
             RangePartitionInfo rangePartitionInfo = (RangePartitionInfo) partitionInfo;
             Set<String> existPartitionNameSet =
                     CatalogChecker.checkPartitionNameExistForAddPartitions(olapTable, singleRangePartitionDescs);
-
+            // partition properties is prior to clause properties
```

Comment:
What is clause property?

Reply:
The clasue property has the form for example:
alter table test2 add partition p11 values less than ("2021-12-24") properties ("storage_medium" = "ssd")
this properties is clasue property because alter table could do batch operation.
if partition is not set property clasue property will apply to set every partition their properties.